cerbero/utils/shell.py

```diff
@@ -267,19 +280,26 @@ def download_curl(url, destination=None, recursive=False, check_cert=True, user=
             cmd += " "
     if not check_cert:
         cmd += "-k "
-    if destination is not None:
-        cmd += "%s -o %s " % (url, destination)
-    else:
-        cmd += "-O %s " % url
+    if not dst:
```

Comment:
in the code below we expect dst to be == destination. For example at line 288.
Probably this code is actually means "if not destination"